tests/client/amo/components/TestAddonDetail.js

```diff
@@ -441,12 +441,15 @@ describe('AddonDetail', () => {
     });
 
     it('prompts you to read many reviews', () => {
+      const i18n = getFakeI18nInst();
       const root = reviewFooterDOM({
         ratingsCount: 5,
+        i18n,
       });
       const footer =
         root.querySelector('.AddonDetail-read-reviews-footer');
       assert.equal(footer.textContent, 'Read all 5 reviews');
+      assert.ok(i18n.ngettext.calledWith('Read %(count)s review', 'Read all %(count)s reviews', 5));
```

Comment:
Instead of this assertion, I think it would be nicer to make a new test that renders with `{ ratingsCount: 1 }` and asserts the value of `footer.textContent`. I know that it was an accident to not call `i18n.ngettext()` in the first place but I don't think adding a regression test for that type of mistake is very helpful.